java/maven/src/org/netbeans/modules/maven/classpath/ClassPathProviderImpl.java

```diff
@@ -370,7 +305,13 @@ private ClassPath getRuntimeClasspath(int type) {
     
     private ClassPath getJava8RunTimeClassPath(int type) {
         final int ftype = type == TYPE_WEB ? TYPE_SRC : type;
-        return computeIfAbsent(JAVA8_RUNTIME_PATH + ftype, () -> ClassPathFactory.createClassPath(ftype == TYPE_SRC ? new RuntimeClassPathImpl(getNBMavenProject()) : new TestRuntimeClassPathImpl(getNBMavenProject(), false)));
+        return computeIfAbsent(JAVA8_RUNTIME_PATH + ftype,
+                () -> ClassPathFactory.createClassPath(
+                        ftype == TYPE_SRC
+                            ? new RuntimeClassPathImpl(getNBMavenProject())
+                            : new TestRuntimeClassPathImpl(getNBMavenProject(), false)
+                )
+        );
```

Comment:
There is no change in code here IIUC? It would be helpful to have stylistic / reformatting changes clearly marked as such and be in separate PRs, rather than intermixed with behavioral changes or refactorings.

Reply:
they are in a separate commit. I am not squashing PRs if they contain a code change commit and a cleanup commit.

I don't feel its needed to create separate PRs in many cases - this does also conserve CI resources.

Reply:
added do-not-squash label